Source/Core/Core/HW/HW.cpp

```diff
@@ -42,6 +43,7 @@ void Init()
   ProcessorInterface::Init();
   ExpansionInterface::Init();  // Needs to be initialized before Memory
   Memory::Init();
+  AddressSpace::Init();
```

Comment:
This initializer depends on `Memory::Init()`

Ideally we should have a comment whenever there are initialization order dependencies.